swabbie-front50/src/test/kotlin/com/netflix/spinnaker/swabbie/front50/ApplicationResourceOwnerResolutionStrategyTest.kt

```diff
@@ -42,16 +41,12 @@ object ApplicationResourceOwnerResolutionStrategyTest {
         Application(name = "test", email = "test@netflix.com")
       )
 
-    subject.resolve(ResourceOne()) shouldMatch equalTo("name@netflix.com")
+    subject.resolve(
+      TestResource(
```

Comment:
Using the `TestResource` class where applicable. Took the opportunity to do a minor cleanup here.